CalibTracker/SiStripLorentzAngle/interface/LA_Filler_Fitter.h

```diff
@@ -4,8 +4,7 @@
 #include <string>
 #include <vector>
 #include <map>
-#include "DataFormats/SiStripDetId/interface/TOBDetId.h"
-#include "DataFormats/SiStripDetId/interface/TIBDetId.h"
+#include "DataFormats/SiStripDetId/interface/SiStripDetId.h"
```

Comment:
indeed, but it shouldn't be included in `TIBorTOBparameters.h` in the first place, IIRC